pkg/keyspace/keyspace.go

```diff
@@ -15,23 +15,42 @@
 package keyspace
 
 import (
+	"context"
+	"crypto/tls"
+	"encoding/binary"
 	"fmt"
+	"strings"
+	"time"
 
+	"github.com/pingcap/kvproto/pkg/keyspacepb"
 	"github.com/pingcap/kvproto/pkg/kvrpcpb"
+	"github.com/pingcap/kvproto/pkg/pdpb"
 	"github.com/pingcap/tidb/pkg/config"
+	"github.com/pingcap/tidb/pkg/util"
+	"github.com/pingcap/tidb/pkg/util/codec"
+	tikvconfig "github.com/tikv/client-go/v2/config"
 	"github.com/tikv/client-go/v2/tikv"
+	pd "github.com/tikv/pd/client"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 )
 
 const (
 	// tidbKeyspaceEtcdPathPrefix is the keyspace prefix for etcd namespace
 	tidbKeyspaceEtcdPathPrefix = "/keyspaces/tidb/"
+
+	// KeyspaceMetaConfigGCManagementType is gc management type in keyspace meta config.
+	KeyspaceMetaConfigGCManagementType = "gc_management_type"
```

Comment:
set `KeyspaceMetaConfigGCManagementType ` and `KeyspaceMetaConfigGCManagementTypeKeyspaceLevelGC ` here, to fix import cycle.